showcase/leakcheck.go

```diff
@@ -44,6 +44,7 @@ var goroutinesToIgnore = []string{
 	"runtime_mcall",
 	"(*loggingT).flushDaemon",
 	"goroutine in C code",
+	"internal/transport/http2_client.go",
```

Comment:
@codyoss this led to a successful local run of `showcase/showcase.bash`, but I'm not sure I have enough background context to understand if this is the right thing to ignore. If this is the accepted change, I'll squash down these commits to get this changeset landed.

Reply:
lgtm. I can squash on merge :)